.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+# See https://pre-commit.com/hooks.html for info on hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-ast
+      - id: check-toml
+      - id: check-yaml
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.3
+    hooks:
+      - id: flake8
+        args: [--max-line-length=88]
+        language_version: python3.7
+
+  - repo: https://github.com/timothycrosley/isort
+    rev: 5.2.0
```

Comment:
The current version is 5.4.1. Is there a reason to not update?

Reply:
No reason other than Timothy has been very busy pushing out new versions. In fact, the current version is now 5.4.**2** 😄  I just pushed a commit that updates to the latest version in both places.